frontend/src/components/shared/additionalInfo/additionalInfo.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../styles/variables.scss';
+
+p {
+  margin: 24px 0 10px;
+}
```

Comment:
this duplicates few times - maybe it is better to add class name and use as global style?